### ISSUE TITLE
Multi View?

### ISSUE THREAD

**Issue by BjoernM85** [WebRTSP/ReStreamer#38]
Hello,

is there a possibility to get a multi view with 4 streams parallel?

Best regards
Björn

**Comment by RSATom** [comment]
Yes it's possible but requires custom/patched  browser client. Maybe you could find something useful in https://github.com/WebRTSP/ReStreamer/issues/7

**Comment by RSATom** [comment]
Also, to simplify use of custom browser clients, some time ago I've added related option to config: https://github.com/WebRTSP/ReStreamer/blob/bf58f741116c179a3f0dedf81bcfe7fa73150bf0/restreamer.conf.sample#L13

**Comment by RSATom** [comment]
it's better late than never... I should be available soon on snap's edge channel.

**Comment by BjoernM85** [comment]
Thanks, but I moved to go2rtc.
